docs/guides/starting-contribution/getting-started.mdx

```diff
@@ -31,32 +31,74 @@ There are 2 ways you can add your profile :
 
 ```markdown github_username.mdx
 ---
-author: Aadarsh Thakur
-github_username: Aadarsh805
+author: Priyankar Pal
+github_username: priyankarpal
 occupation: Student
+github: https://github.com/priyankarpal
+twitter: https://twitter.com/priyankarpal
+linkedin: https://www.linkedin.com/in/priyankarpal
 ---
 
 # Introduction
 
-Hi I'm Aadarsh Thakur, a frotend developer from Indore, India. I am currently a 3rd year Computer Science student persuing B-Tech.
-I love building cool projects, contributing to OpenSource and getting involved in awesome tech communities.
+Hi👋 there! It's Priyankar Pal, a First-year Computer Science Engineering student & a Tech Enthusiast. I'm a Front-end Developer.I am always looking for ways to contribute to the open-source community and positively impact the tech industry.
 
 ## Skills
 
-<Skills>
-  <Skill
-    name="Next Js"
-    icon={/* svg icon for nextJS */}
-  >
-   NextJS
-  </Skill>
-  <Skill
-    name="Typescript"
-    icon={/*  svg icon for Typescript  */}
-  >
-    Typescript
-  </Skill>
-</Skills>
+<Tabs>
+
+<Tab title="Frame Work">
+  <Skills title="Frame Works">
+    <Skill
+      name="React"
+      href="https://reactjs.org/"
+      icon=
+      {
+        <svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This would be overwhelming for newcomers, instead write `paste svg code for icon` mention simple icons as an example where they can copy svg code easily